andela/ah-fortem-frontend
# 166980313 Allows users to view tags and filter articles by tags

#### Title

This PR allows users to view tags and filter articles by tags
#### Description



As a User, I want to be able to view various article tags to enable easy filtering and navigation to articles of interest

#### Type of change








#### How Has This Been Tested?
#### Acceptance Criteria: 



#### How Can This Be Manually Tested?
To be able to test the feature, check out the branch `story/166980313-view-tags-filter-by-tags` and then `npm run dev`.
Click on any of the tags to filter
![image](https://user-images.githubusercontent.com/44572764/60900025-2dcfa600-a274-11e9-999e-0f6381ce4f20.png)


#### Checklist:
- [x] Lists all tags
- [x] Allows user to filter articles by tags
- [x] Adds tests

#### PT stories
[#166980313](https://www.pivotaltracker.com/story/show/166980313)
